Month1/Week2/Section1/Exercise3/Main3.cpp

```diff
@@ -135,16 +139,21 @@ int main()
 	std::cout << "Your choice: " << Choice << std::endl;
 	if (Choice == 1) 
 	{
-		FCircle circle;
-		circle.InPutDataCircle();
-		circle.GetAreaCicle();
-		circle.GetPerimeterCircle();
+		FCircle Circle;
+		Circle.InputDataCircle();
+		Circle.GetAreaCicle();
```

Comment:
Why call GetAreaCircle and GetPerimeterCircle here if you're not doing anything with those values?

Reply:
Unused GetArea and GetPerimeter Eliminated